src/kudu/consensus/metadata.proto

```diff
@@ -135,6 +175,16 @@ message RaftConfigPB {
 
   // The set of peers in the configuration.
   repeated RaftPeerPB peers = 3;
+
```

Comment:
its probably not a good idea to keep configs which we are not going to get consensus on within RaftConfigPB
RaftConfigPB has an opid_index to mark its position in log and is stored in consensus metadata.
Its primary purpose is to have the ring members.

Reply:
This is exactly what I said before. How would you persist them?

Reply:
Repurposed the commit for creating analogues of thrift structs on plugin that will be populated in server options. Persistence shall be deferred.